docs/source/examples/checks/integrity/data_duplicates.ipynb

```diff
@@ -5,19 +5,28 @@
    "id": "06bdcdfc",
```

Comment:
<p>This notebooks provides an overview for using and understanding the <strong>data duplicates </strong>check.</p><p> </p><p>-</p><p>The "Why data duplicates?" link doesn't work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>